src/System.Private.CoreLib/shared/System/Enum.cs

```diff
@@ -493,7 +493,7 @@ private static bool TryParse(Type enumType, string? value, bool ignoreCase, bool
 
                 default:
                     parsed = TryParseRareEnum(rt, value, valueSpan, ignoreCase, throwOnFailure, out object? objectResult);
-                    result = parsed ? (TEnum)objectResult! : default; // TODO-NULLABLE: https://github.com/dotnet/roslyn/issues/34976
+                    result = parsed ? (TEnum)objectResult! : default;
```

Comment:
Didn't bother to annotate TryParseRareEnum because it is private, right?

Reply:
Actually, I just didn't notice the `out object? result`.  I'll add NotNullWhen(true) to it.